### PR TITLE
FEATURES: Facebook, Timer and Wallpaper

### PR DESCRIPTION
**Pull request by ariv797**
Jarvis can now get your Facebook notifications, change your wallpaper and set you a timer

NOTE: This uses Unsplash api key
           https://unsplash.com/developers
           This uses Facebook's GraphAPI key
           https://towardsdatascience.com/how-to-use-facebook-graph-api-and-extract-data-using-python-1839e19d6999

### CODE REVIEW

**Comment by ariv797** [comment]
Can you please try to add Google Calendar support as I believe it is available for both MacOs and Windows

File: jarvis.py [hunk 1]
```diff
@@ -2520,6 +2531,67 @@ def vpn_checker():
                     "integrations will work with VPN enabled.")
     else:
         return '.'.join(ip_address.split('.')[0:3])
+                         
+                         
+def timer(converted):
+    speaker.say("For how many minutes?")
+    speaker.runAndWait()
+    timer0 = listener(3, 5)
+    converted.replace('for', 'minutes', 'minute', 'in', '')
```
Comment (thevickypedia) [inline jarvis.py:2540]: refer docs on how .replace() works

File: jarvis.py [hunk 2]
```diff
@@ -2520,6 +2531,67 @@ def vpn_checker():
...
+
+    timer1 = float(timer0)
+    timer2 = timer1 * 60
+    speaker.say(f'I will remind you in {timer2} seconds')
```
Comment (thevickypedia) [inline jarvis.py:2544]: This loop will not run until the timer finishes

File: jarvis.py [hunk 3]
```diff
@@ -2520,6 +2531,67 @@ def vpn_checker():
...
+
+    time.sleep(timer2)
```
Comment (thevickypedia) [inline jarvis.py:2546]: Jarvis will be unusable until timer finishes

File: jarvis.py [hunk 4]
```diff
@@ -2520,6 +2531,67 @@ def vpn_checker():
...
+    speaker.say('Your time has been finished sir')
+                         
+                         
+def facebook(profile):
```
Comment (thevickypedia) [inline jarvis.py:2550]: I have this as a separate project but `GraphAPI` doesn't work the way you expect, the APIs are not persistent and keeps expiring often, so I will not be integrating `facebook` anytime soon

File: jarvis.py [hunk 5]
```diff
@@ -2520,6 +2531,67 @@ def vpn_checker():
...
+    updates = []
+    for notification in results['data']:
+        updates.append(notification['title'])
+
+    count = len(results['data'])
+    speaker.say("You have " + str(count) + " Facebook notifications. " + " ".join(updates) + ". ")
+    return
+                         
+           
+def wallpaper():
```
Comment (thevickypedia) [inline jarvis.py:2575]: I'm not sure what value does changing wallpaper provide with the cost involved. FYI - by cost, I mean the effort, time complexity and additional modules. Rather, you can use windows' powershell or apple script to do the task for free. And absolutely NO print statements for J

**Pull request closed without merging**